README.md

```diff
@@ -45,14 +48,17 @@ type Position struct {
 The connector goes through two reading modes.
 
 * Snapshot mode (Value 0): which loops through the GCS bucket and returns the objects that are already in there. The _position type_ during this mode is 0. which makes the connector know at what mode it is and what object it last
-  read. The _position Timestamp_ will be used when changing to CDC mode, the iterator will capture changes that
-  happened after that.
+ read. The _position Timestamp_ will be used when changing to CDC mode, the iterator will capture changes that
+ happened after that.
 
 * CDC mode: (Value 1) this mode iterates through the GCS bucket every `pollingPeriod` and captures new actions made on the bucket.
-  the _Position Type_ during this mode is 1. This position is used to return only the
-  actions with a _Position Timestamp_ higher than the last record returned, which will ensure that no duplications are in
-  place.
+ the _Position Type_ during this mode is 1. This position is used to return only the
+ actions with a _Position Timestamp_ higher than the last record returned even if the timestamp got matched then the decision would be based on lexicographical comparison, which will ensure that no duplications are in
+ place.
 
+ The CDC mode will start after a single action(Insert/Update/Delete) completed post snapshot.
```

Comment:
does it start CDC after the first change? or after the last record from snapshot? we should have an integration test for (if the connector stops after snapshot and starts again.. it should start from CDC)

Reply:
GCS object iterator doesn't support the HasNext function to check whether the there is a next object. So in the current implementation CDC will start after the first change.

Let me know explore are there any options to support "if the connector stops after snapshot and starts again.. it should start from CDC"

Reply:
@maha-hajja  There is no direct method provide from the GCS API to know the last item. But i went through the internal implementation 
when PageInfo.Token is empty and PageInfo.Remaining() is zero then that is the last item of the snapshot iterator. so where we can switch it back to CDC.
I think will track this issue separately. Pls share your observations

Reference Doc:
https://pkg.go.dev/google.golang.org/api/iterator#PageInfo
https://pkg.go.dev/google.golang.org/api/iterator#PageInfo.Remaining

Reply:
Issue #23